pkg/lint/lintersdb/manager.go

```diff
@@ -275,6 +277,9 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		linter.NewConfig(golinters.NewExportLoopRef()).
 			WithPresets(linter.PresetBugs).
 			WithURL("https://github.com/kyoh86/exportloopref"),
+		linter.NewConfig(golinters.NewExhaustive(exhaustiveCfg)).
+			WithPresets(linter.PresetBugs).
+			WithURL("https://github.com/nishanths/exhaustive"),
```

Comment:
add `WithLoadForGoAnalysis()` please - otherwise if only this linter is enabled it won't work

Reply:
@all also, we should automate this check: it's really very common fault.

Reply:
Thanks for explaining, I've added it.